test/gui/tst_syncing/test.feature

```diff
@@ -353,4 +353,13 @@ Feature: Syncing files
         Then as "Alice" folder "folder1" should exist on the server
         And as user "Alice" folder "folder1" should contain "500" items on the server
         And as "Alice" folder "folder2" should exist on the server
-        And as user "Alice" folder "folder2" should contain "500" items on the server
\ No newline at end of file
+        And as user "Alice" folder "folder2" should contain "500" items on the server
+
+
+    Scenario: Choosing local download directory in advanced configuration
+        Given the user has started the client
+        When the user adds the following account information:
+            | server   | %local_server% |
+            | user     | Alice          |
+            | password | 1234           |
```

Comment:
this step should already exist. Or it was removed in refactoring. @saw-jan

Reply:
We have `Given` step but not `When` I guess